lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -0,0 +1,49 @@
+class ActiveModel::Serializer::Adapter::Attributes < ActiveModel::Serializer::Adapter
+        def serializable_hash(options = nil)
+          options ||= {}
+          if serializer.respond_to?(:each)
+            serializer.map { |s| Attributes.new(s).serializable_hash(options) }
+          else
+            hash = {}
+
+            core = cache_check(serializer) do
+              serializer.attributes(options)
+            end
+
+            serializer.associations.each do |association|
```

Comment:
I see that all this came from the json adapter.
I think I'm getting PRs mixed in my head. bah.

it'll be interesting to see how this plays in to the nested associations stuff.